pkg/dao/mock_dao.go

```diff
@@ -47,6 +47,11 @@ func (_m *MockDao) BatchGetSpecs(_a0 string) ([]*apb.Spec, error) {
 	return r0, r1
 }
 
+// BatchGetBundleInstances a noop
+func (_m *MockDao) BatchGetBundleInstances() ([]*apb.ServiceInstance, error) {
+	return nil, nil
```

Comment:
I think that this is not correct. I think you may need to re-generate this mock. https://github.com/vektra/mockery

Reply:
run `make generate` and it will do it iirc :)

Reply:
I have generated tests using mockery in subsequent [commit](https://github.com/openshift/ansible-service-broker/pull/1008/commits/35929277d4fa1e8539c592f9add0727dcf07f44e)

Reply:
done. @jmrodri I ran `make generate` but it imported `apb` as `bundle`, renamed that to `apb` to reduce number of lines in git diff.